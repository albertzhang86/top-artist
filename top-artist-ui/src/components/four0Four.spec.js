import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import Four0Four from './four0Four.js';

describe('Four0Four', () => {
    it('should render correctly', () => {
        const component = create(
            <Four0Four/>
        );
        expect(component).toMatchSnapshot();
    });
});
