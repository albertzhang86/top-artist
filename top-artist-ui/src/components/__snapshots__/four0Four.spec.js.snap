// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Four0Four should render correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-xs-offset-0 col-sm-offset-2 col-xs-12 col-sm-8"
  >
    <h3>
      Page Not Found
    </h3>
  </div>
</div>
`;
